common/lib/common-utils/src/bufferBrowser.ts

```diff
@@ -4,7 +4,7 @@
  */
 
 import * as base64js from "base64-js";
-import { assert } from "./assert";
+import { Uint8ArrayToArrayBuffer } from "./indexNode";
```

Comment:
bufferBrowser needs to have its own implementation and shouldn't use any node implementations

Reply:
Done